orunmila.go

```diff
@@ -268,16 +268,12 @@ func createDbFileifNotExists(dbPtr string) {
 	}
 }
 
-// parse args of the import subcommand and exec it
-func vacuumSubcmd(args []string) {
-	flag := flag.NewFlagSet("vacuum", flag.ContinueOnError)
-
-	flag.Parse(args)
-
-	createDbFileifNotExists(*dbPtr)
+// parse args of the vaccum subcommand and exec it
+func vacuumSubcmd() {
+	// parse the global args instead (debug && db path)
+	flag.Parse()
```

Comment:
if there are no args do we still need to flag.parse()?

Reply:
also usage here maybe?

Reply:
usage will be on it's way, same note about the `flag.Parse()` call, it's removed now